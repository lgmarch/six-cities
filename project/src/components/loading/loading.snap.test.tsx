import {render} from '@testing-library/react';
import Loading from './loading';

describe('Component: Loading', () => {
  test('should render correctly', () => {
    const {container} = render(<Loading />);
    expect(container).toMatchSnapshot();
  });
});
