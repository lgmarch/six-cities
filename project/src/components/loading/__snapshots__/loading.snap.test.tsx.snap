// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Loading should render correctly 1`] = `
<div>
  <p>
    Loading ...
  </p>
</div>
`;
